### ISSUE TITLE
Video unavailable. Only on tor

### ISSUE THREAD

**Issue by coolcoderplus** [omarroth/invidious#816]
When I log in without tor on invidio.us, everything is fine I can access my favorites and play videos without any problem.
But when i use : axqzx4s6s54s32yentfqojs3x5i7faxza6xo3ehd4bzzsg2ii4fv2iid.onion
It is then impossible for me to play a single video to which I have subscribed.
And I have the following message on all the videos : Video unavailable.
I appreciate the tor service, and I only use it :)
there's certainly a configuration error somewhere.

**Comment by Booteille** [comment]
Hi.
Your issue is a duplicate of #811. The IP of the Tor instance is blocked by Google.

Please, continue the discussion on the other thread instead.

**Comment by coolcoderplus** [comment]
> Hi.
> Your issue is a duplicate of #811. The IP of the Tor instance is blocked by Google.
> 
> Please, continue the discussion on the other thread instead.

Ok thanks Booteille

**Comment by omarroth** [comment]
As @Booteille mentioned this is caused by Google. I've rebuilt the tor instance on another machine, which appears to have resolved the issue.

**Comment by omarroth** [comment]
Closing in favor of #811.
